### ISSUE TITLE
[BUG]Can't get it to work. Error 503 in Backend. 

### ISSUE THREAD

**Issue by zzzooorrrx** [eliashaeussler/typo3-form-consent#55]
### TYPO3 version

11.5.10

### PHP version

8.1.5

### Extension version

0.5.0

### Composer mode

- [X] I'm running TYPO3 in composer mode.

### Operating system

macOS 11.6.4

### Current behavior

Hi,
I already had an older version of form_consent installed. After failing migration to 0.5.0 I deinstalled everything. 
And, of course, empty whole chaches.

After installing again, adding Extension template and build the first simple trial.
In Backend, when try to view the result in Backend Preview Mode I get an 503-error:

503
Oops, an error occurred!
The finisher preset identified by "Consent" could not be found, or the implementationClassName was not specified.
More information regarding this error might be available [online](https://typo3.org/go/exception/CMS/1328709784).

In Frontend I also get an error after submitting.

The yaml-file is the following:

```yaml
renderingOptions:
  submitButtonLabel: Absenden
type: Form
identifier: testmitconsent
label: testmitconsent
prototypeName: standard
finishers:
  -
    options:
      subject: 'Bestätigung Ihrer Eintragung'
      recipientAddress: '{email-1}'
      recipientName: '{text-2}'
      senderAddress: ''
      senderName: ''
      approvalPeriod: '86400'
      showDismissLink: false
      confirmationPid: '101'
      storagePid: '99'
    identifier: Consent
renderables:
  -
    renderingOptions:
      previousButtonLabel: 'Vorherige Seite'
      nextButtonLabel: 'Nächster Schritt'
    type: Page
    identifier: page-1
    label: Step
    renderables:
      -
        defaultValue: ''
        type: Text
        identifier: text-1
        label: 'Dieses Formular testet den consent'
      -
        defaultValue: ''
        type: Email
        identifier: email-1
        label: 'E-Mail Adresse'
        properties:
          fluidAdditionalAttributes:
            required: required
          validationErrorMessages:
            -
              code: 1221560910
              message: 'Bitte Email-Adresse eingeben'
            -
              code: 1221560718
              message: 'Bitte Email-Adresse eingeben'
            -
              code: 1347992400
              message: 'Bitte Email-Adresse eingeben'
            -
              code: 1347992453
              message: 'Bitte Email-Adresse eingeben'
            -
              code: 1221559976
              message: 'Bitte überprüfen Sie die Email-Adresse'
        validators:
          -
            identifier: EmailAddress
          -
            identifier: NotEmpty
      -
        defaultValue: ''
        type: Text
        identifier: text-2
        label: 'Vor- und Nachname'
        properties:
          fluidAdditionalAttributes:
            required: required
          validationErrorMessages:
            -
              code: 1221560910
              message: 'bitte geben Sie Ihren Vor- und Nachnamen ein!'
            -
              code: 1221560718
              message: 'bitte geben Sie Ihren Vor- und Nachnamen ein!'
            -
              code: 1347992400
              message: 'bitte geben Sie Ihren Vor- und Nachnamen ein!'
            -
              code: 1347992453
              message: 'bitte geben Sie Ihren Vor- und Nachnamen ein!'
        validators:
          -
            identifier: NotEmpty
```

thx for help!

### Expected behavior

Preview the form in backend preview mode in form module

After submit I expect to receive an consent email with link

### Additional context

I'm running TYPO3 in classic mode. 
i have to check the composer box, optherwise i couldn't submit this bug report.

Thx for your help!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

**Comment by eliashaeussler** [comment]
Hi @zzzooorrrx, thanks for your report.

Did you include the TypoScript template as written at https://github.com/eliashaeussler/typo3-form-consent#open_file_folder-configuration?

**Comment by eliashaeussler** [comment]
@zzzooorrrx I was able to reproduce the problem when deactivating the extension in extension manager. Can you check if the extension is active?

**Comment by zzzooorrrx** [comment]
@eliashaeussler 
> Did you include the TypoScript template
Yes it is included by static templates from extensions. 
Tried also by adding it manually into the TS setup.

> I was able to reproduce the problem when deactivating the extension in extension manager. Can you check if the extension is active?
It is active. 
Also deactivated it and activated it again.
Same behaviour on my side.

**Comment by eliashaeussler** [comment]
> In Backend, when try to view the result in Backend Preview Mode I get an 503-error
>
> 503
> Oops, an error occurred!
> The finisher preset identified by "Consent" could not be found, or the implementationClassName was not specified.

This is indeed a bug which I already fixed with #56. Will be released in a few minutes. Please update to 0.5.1 then.

> In Frontend I also get an error after submitting.

This should be solved by including the TS template at `EXT:form_consent/Configuration/TypoScript`. If this does not work for you, please create a new issue and post the exact error message.
